project.clj

```diff
@@ -39,6 +39,7 @@
                                org.apache.httpcomponents/httpclient
                                net.sourceforge.nekohtml/nekohtml
                                ring/ring-core]]
+                 [com.cemerick/url "0.1.1"]                           ; URL library
```

Comment:
`-site-url` is guaranteed to never have a trailing slash, the function that sets it already strips off trailing slashes